From 15b41be88214b557cdc7a35b39f0e3a5eb2bcd0d Mon Sep 17 00:00:00 2001
From: Hu Kejun <william.hu@rock-chips.com>
Date: Wed, 15 Aug 2018 14:09:05 +0800
Subject: [PATCH] media: rockchip: isp1: fix two dvp sensor can't connect to
 rkisp1 sink pad

when two dvp sensor connect to rkisp1, only one sensor can connect to sink
pad, another sensor connect to sink params pad.

Change-Id: I6cc9e6db3ad074c82c15a9cc642330ece6997d09
Signed-off-by: Hu Kejun <william.hu@rock-chips.com>
---
 drivers/media/platform/rockchip/isp1/dev.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/media/platform/rockchip/isp1/dev.c b/drivers/media/platform/rockchip/isp1/dev.c
index aa50ea969971..97d49322de17 100644
--- a/drivers/media/platform/rockchip/isp1/dev.c
+++ b/drivers/media/platform/rockchip/isp1/dev.c
@@ -299,7 +299,7 @@ static int rkisp1_create_links(struct rkisp1_device *dev)
 		ret = media_entity_create_link(
 				&sensor->sd->entity, pad,
 				&dev->isp_sdev.sd.entity,
-				RKISP1_ISP_PAD_SINK + s,
+				RKISP1_ISP_PAD_SINK,
 				s ? 0 : MEDIA_LNK_FL_ENABLED);
 		if (ret) {
 			dev_err(dev->dev,
-- 
2.35.3

